### ISSUE TITLE
 authorized SP API

### ISSUE THREAD

**Issue by csx-bill** [amzn/selling-partner-api-docs#1728]
Hello, after I successfully authorized SP API, I will prompt this error when calling the order API interface

Roles may not be assumed by root accounts. (Service: AWSSecurityTokenService; Status Code: 403; Error Code: AccessDenied; Request ID: a897b09d-31b5-40bf-9e57-361146de68c7)

**Comment by stefnats** [comment]
Are using Role or User ARN in your application? If you are using Role ARN, you need to AssumeRole first, as user you don't.

**Comment by csx-bill** [comment]
> 在您的应用程序中使用角色或用户 ARN？如果您正在使用角色 ARN，则需要首先 AssumeRole，作为用户，您不需要。
My North America authorization is successful. Pull the API and Far East will prompt this error

**Comment by stefnats** [comment]
Sorry, i don't understand your answer.

**Comment by csx-bill** [comment]
> North America
My North America authorization was successful
